pkg/server/server_test.go

```diff
@@ -1383,3 +1383,33 @@ func TestDefaultMaxConcurrentReadSettings(t *testing.T) {
 	require.EqualValues(t, math.MaxUint32, s.maxConcurrentReadsForCheck)
 	require.EqualValues(t, math.MaxUint32, s.maxConcurrentReadsForListObjects)
 }
+
+func TestWriteAuthorizationModelWithExperimentalEnableModularModels(t *testing.T) {
+	ctx := context.Background()
+	storeID := ulid.Make().String()
+
+	mockController := gomock.NewController(t)
+	defer mockController.Finish()
+
+	mockDatastore := mockstorage.NewMockOpenFGADatastore(mockController)
+
+	s := MustNewServerWithOpts(
+		WithDatastore(mockDatastore),
+	)
+
+	t.Run("rejects_request_with_schema_version_1.2", func(t *testing.T) {
```

Comment:
Please add one more with enables it using the `With..` option and make sure it passes at the server level